core/document-types/page-base.php

```diff
@@ -171,6 +171,29 @@ public static function register_style_controls( $document ) {
 		Plugin::$instance->controls_manager->add_custom_css_controls( $document );
 	}
 
+	public static function get_labels() : array {
+		$plural_label   = static::get_plural_title();
+		$singular_label = static::get_title();
+
+		$labels = [
+			'name' => $plural_label, // Already translated.
+			'singular_name' => $singular_label, // Already translated.
+			'all_items' => sprintf( __( 'All %s', 'elementor' ), $plural_label ),
+			'add_new' => esc_html__( 'Add New', 'elementor' ),
+			'add_new_item' => sprintf( __( 'Add New %s', 'elementor' ), $singular_label ),
+			'edit_item' => sprintf( __( 'Edit %s', 'elementor' ), $singular_label ),
+			'new_item' => sprintf( __( 'New %s', 'elementor' ), $singular_label ),
+			'view_item' => sprintf( __( 'View %s', 'elementor' ), $singular_label ),
+			'search_items' => sprintf( __( 'Search %s', 'elementor' ), $plural_label ),
+			'not_found' => sprintf( __( 'No %s found.', 'elementor' ), strtolower( $plural_label ) ),
+			'not_found_in_trash' => sprintf( __( 'No %s found in Trash.', 'elementor' ), strtolower( $plural_label ) ),
+			'parent_item_colon' => '',
+			'menu_name' => $plural_label,
+		];
+
+		return $labels;
+	}
```

Comment:
@nicoladj77 Why did you have to add this here?

Reply:
@louiswol94 this is to streamline adding CPTs in modules. It's always the same so I thought it mades sense to have it here.

Reply:
It feels risky :)